cmd/forwarder/http.go

```diff
@@ -35,38 +35,46 @@ func NewPayload(sa string, ri string, b []byte) payload {
+	Config                IssConfig
+	FixerFunc             FixerFunc
+	shutdownCh            shutdownCh
+	deliverer             deliverer
+	isShuttingDown        bool
+	auth                  authenticater.Authenticater
+	posts                 metrics.Timer   // tracks metrics about posts
+	healthChecks          metrics.Timer   // tracks metrics about health checks
+	pErrors               metrics.Counter // tracks the count of post errors
+	pSuccesses            metrics.Counter // tracks the number of post successes
+	pAuthErrors           metrics.Counter // tracks the count of auth errors
+	pAuthSuccesses        metrics.Counter // tracks the number of auth successes
+	pMetadataLogsReceived metrics.Counter // tracks the number of logs that have metadata that have been received
+	pLogsReceived         metrics.Counter // tracks the number of logs that have been received
+	pMetadataLogsSent     metrics.Counter // tracks the number of logs that have metadata that have been received
+	pLogsSent             metrics.Counter // tracks the number of logs that have been received
 	sync.WaitGroup
 }
 
 func newHTTPServer(config IssConfig, auth authenticater.Authenticater, fixerFunc FixerFunc, deliverer deliverer) *httpServer {
 	return &httpServer{
-		auth:           auth,
-		Config:         config,
-		FixerFunc:      fixerFunc,
-		deliverer:      deliverer,
-		shutdownCh:     make(shutdownCh),
-		posts:          metrics.GetOrRegisterTimer("log-iss.http.logs", config.MetricsRegistry),
-		healthChecks:   metrics.GetOrRegisterTimer("log-iss.http.healthchecks", config.MetricsRegistry),
-		pErrors:        metrics.GetOrRegisterCounter("log-iss.http.logs.errors", config.MetricsRegistry),
-		pSuccesses:     metrics.GetOrRegisterCounter("log-iss.http.logs.successes", config.MetricsRegistry),
-		pAuthErrors:    metrics.GetOrRegisterCounter("log-iss.auth.errors", config.MetricsRegistry),
-		pAuthSuccesses: metrics.GetOrRegisterCounter("log-iss.auth.successes", config.MetricsRegistry),
-		isShuttingDown: false,
+		auth:                  auth,
+		Config:                config,
+		FixerFunc:             fixerFunc,
+		deliverer:             deliverer,
+		shutdownCh:            make(shutdownCh),
+		posts:                 metrics.GetOrRegisterTimer("log-iss.http.logs", config.MetricsRegistry),
+		healthChecks:          metrics.GetOrRegisterTimer("log-iss.http.healthchecks", config.MetricsRegistry),
+		pErrors:               metrics.GetOrRegisterCounter("log-iss.http.logs.errors", config.MetricsRegistry),
+		pSuccesses:            metrics.GetOrRegisterCounter("log-iss.http.logs.successes", config.MetricsRegistry),
+		pAuthErrors:           metrics.GetOrRegisterCounter("log-iss.auth.errors", config.MetricsRegistry),
+		pAuthSuccesses:        metrics.GetOrRegisterCounter("log-iss.auth.successes", config.MetricsRegistry),
+		pMetadataLogsReceived: metrics.GetOrRegisterCounter("log-iss.metadata_logs.received", config.MetricsRegistry),
```

Comment:
Pendantic, but the idiomatic pattern seems to be - over _ in metrics. Also an personal preference. 🤷🏻‍♂️ Same for the one below, but not going to duplicate comments. 

Ultimately your call.